source/common/common/standard/logger_impl.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace Envoy {
```

Comment:
I think since this implicitly depends on base_logger.h for a definition of StandardLogger, it should probably include it (and have a dep in the BUILD file).